### PR TITLE
Add shortlink panel to block editor

### PR DESCRIPTION
**Pull request by florianbrinkmann**
This PR adds a shortlink panel to the block editor where one can copy the shortlink via a button.

Currently, the PR includes the build JS files, so that it will continue to work with just downloading the repo code. If that is not wanted, we could also try to add the build process to the GitHub workflow, so we do not need the build files in the repo.

### CODE REVIEW

**Comment by pfefferle** [comment]
Thanks a lot! Seems to work like a charm!

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+			text={ humEditorObject.shortlink }
+			onCopy={ () => setState( { hasCopied: true } ) }
+			onFinishCopy={ () => setState( { hasCopied: false } ) }
+		>
+			{ hasCopied ? humEditorObject.copyButtonCopiedLabel : humEditorObject.copyButtonLabel }
+		</ClipboardButton>
+	</PluginDocumentSettingPanel>
+) );
+
+registerPlugin( 'plugin-document-setting-panel-demo', {
```
Comment (pfefferle) [inline src/index.js:35]: this seems to be a demo label ;)

Comment (florianbrinkmann) [reply src/index.js:35]: Oh yes :D Will fix it

Comment (florianbrinkmann) [reply src/index.js]: Fixed :)

**Review by pfefferle — APPROVED** [review]
LGTM! Thanks a lot @florianbrinkmann

**Comment by florianbrinkmann** [comment]
You’re welcome @pfefferle :)

**Pull request merged**